packages/cashier/src/modules/cashier-onboarding/components/cashier-onboarding-p2p-card/cashier-onboarding-p2p-card.tsx

```diff
@@ -15,12 +15,12 @@ const CashierOnboardingP2PCard: React.FC = observer(() => {
     const { openRealAccountSignup } = ui;
     const { setDepositTarget } = general_store;
     const history = useHistory();
+    const { data: has_p2p_supported_currencies } = useHasP2PSupportedCurrencies();
     const { data: is_p2p_enabled } = useIsP2PEnabled();
-    const has_usd_currency = useHasUSDCurrency();
     const has_fiat_currency = useHasFiatCurrency();
-    const should_show_p2p_card = is_p2p_enabled || has_usd_currency;
     const can_switch_to_fiat_account = is_crypto() && has_fiat_currency;
     const [is_dialog_visible, setIsDialogVisible] = useState(false);
+    const should_show_p2p_card = is_p2p_enabled || (is_crypto() && has_p2p_supported_currencies);
```

Comment:
```suggestion
    const should_show_p2p_card = is_p2p_enabled || has_p2p_supported_currencies;
```
@farhan-nurzi-deriv Do we need that `is_crypto` check? 🤔